runtime/queries/metricsview_comparison_toplist_test.go

```diff
@@ -19,12 +19,18 @@ import (
 	_ "github.com/rilldata/rill/runtime/drivers/duckdb"
 )
 
+// replace with "ad_bids_druid" to test on druid
+const AdBidsProject = "ad_bids"
+
+// replace with "timestamp" to test on druid
+const AdBidsTimestamp = "timestamp"
```

Comment:
The test scaffolding around the analytical APIs is already pretty complex, so I don't think having constants to comment/uncomment in multiple different files is the way to add Druid support to them.

It would be really nice to add Druid support, but I think we should do it in a separate PR and figure out a proper test abstraction that would allow us to test both Druid and DuckDB in CI/CD. We should also have Druid started using Testcontainers similar to the test cases in `runtime/drivers`.

Reply:
Got it.